lib/lotus/view/rendering/partial_finder.rb

```diff
@@ -34,14 +35,24 @@ class PartialFinder < TemplateFinder
         # @since 0.4.3
         # @api private
         def find
-          if path = partial_template_under_view_path
-            View::Template.new(path, @view.configuration.default_encoding)
-          else
-            super
-          end
+          find_cached_template
         end
 
         protected
+
+        # @since x.x.x
+        # @api private
+        def find_cached_template
```

Comment:
@stevehook This abstraction is not needed because `#find` only invokes it and don't do anything else. So `#find_cached_template` can become `#find` instead.